zhaquirks/tuya/ts0601_trv.py

```diff
@@ -1840,18 +1852,18 @@ def __init__(self, *args, **kwargs):
         translation_key="child_lock",
         fallback_name="Child lock",
     )
-    .tuya_switch(
-        dp_id=35,
-        attribute_name="frost_protection",
-        translation_key="frost_protection",
-        fallback_name="Frost protection",
-    )
     .tuya_switch(
         dp_id=39,
         attribute_name="scale_protection",
         translation_key="scale_protection",
         fallback_name="Scale protection",
     )
+    .tuya_switch(
+        dp_id=36,
+        attribute_name="frost_protection",
+        translation_key="frost_protection",
+        fallback_name="Frost protection",
+    )
```

Comment:
You can swap this back to keep in order of DP id. Can't add a suggestion on this.